modules/tinymce/src/plugins/lists/test/ts/browser/IndentTest.ts

```diff
@@ -448,4 +448,43 @@ describe('browser.tinymce.plugins.lists.IndentTest', () => {
 
     assert.equal(editor.selection.getNode().nodeName, 'LI');
   });
+
+  context('Parent context', () => {
+    const testCommandAtTextPath = (command: string) => (inputHtml: string, path: number[], expectedHtml: string) => () => {
+      const editor = hook.editor();
+      editor.setContent(inputHtml);
+
+      TinySelections.setCursor(editor, path, 0);
+      editor.execCommand(command);
+
+      TinyAssertions.assertContent(editor, expectedHtml);
+    };
+
+    const testIndentAtTextPath = testCommandAtTextPath('Indent');
+    const testOutdentAtTextPath = testCommandAtTextPath('Outdent');
+
+    it('TINY-8068: indent list item inside div inside list item', testIndentAtTextPath(
```

Comment:
Similar to the changelog should this and other tests in this file be TINY-7209 instead?